app/instrumentation-tests/src/org/commcare/androidTests/DeveloperOptionsTests.kt

```diff
@@ -0,0 +1,132 @@
+import org.junit.Test
+import org.junit.runner.RunWith
+
+
+@RunWith(AndroidJUnit4::class)
+@LargeTest
+@BrowserstackTests
+class DeveloperOptionsTests : BaseTest() {
+
+    companion object {
+        const val CCZ_NAME = "basic_tests.ccz"
+        const val APP_NAME = "Basic Tests"
+    }
+
+    @Before
+    fun setup() {
+        installApp(APP_NAME, CCZ_NAME)
+        InstrumentationUtility.login("test1", "123")
+    }
+
+    @After
+    fun tearDown() {
+        InstrumentationUtility.logout()
+    }
+
+    @Test
+    fun testDevelopersOptions() {
+        InstrumentationUtility.enableDeveloperMode()
+        InstrumentationUtility.openOptionsMenu()
+        onView(withText("Settings"))
+            .perform(click())
+        onView(withText("Developer Options"))
+            .perform(click())
+        onView(withText("Developer Mode Enabled")).isPresent()
+        onView(withText("Show Update Options Item")).isPresent()
+        onView(withText("Image Above Question Text Enabled")).isPresent()
+        onView(withText("Detect and Trigger Purge on Form Save")).isPresent()
+        InstrumentationUtility.gotoHome()
+        testUpdateOptions()
+        testImageAboveText()
+        testImageBelowText()
+    }
+
+    fun testUpdateOptions(){
+        InstrumentationUtility.openOptionsMenu()
+        onView(withText("Settings"))
+            .perform(click())
+        onView(withText("Developer Options"))
+            .perform(click())
+        onView(withText("Show Update Options Item"))
+            .perform(click())
+        onView(withText("Enabled"))
+            .perform(click())
+        InstrumentationUtility.gotoHome()
+        InstrumentationUtility.openOptionsMenu()
+        onView(withText("Settings"))
+            .perform(click())
+        onView(withText("Update Options"))
+            .perform(click())
+        onView(withText("Latest saved state"))
+            .perform(click())
+        InstrumentationUtility.gotoHome()
+        InstrumentationUtility.openOptionsMenu()
+        onView(withText("Update App"))
+            .perform(click())
+        InstrumentationUtility.waitForView(withText("App is up to date"))
+        onView(withText("App is up to date")).isPresent()
+        InstrumentationUtility.gotoHome()
+    }
+    fun testImageAboveText(){
+        InstrumentationUtility.openOptionsMenu()
+        onView(withText("Settings"))
+            .perform(click())
+        onView(withText("Developer Options"))
+            .perform(click())
+        onView(withText("Image Above Question Text Enabled"))
+            .perform(click())
+        onView(withText("Enabled"))
+            .perform(click())
+        InstrumentationUtility.gotoHome()
+        InstrumentationUtility.openForm(0,0)
+        onView(withId(R.id.image)).check(isAbove(withSubstring("Enter a Name")))
+        onView(withClassName(endsWith("EditText"))).perform(typeText("Image above text"))
+        InstrumentationUtility.submitForm()
```

Comment:
So I think we can simply exit the form without submitting here as that doesn't test anything related to this test. Also the same comment for `testImageBelowText`

Reply:
replaced `submitForm()` calls with `exitForm()`